features/addressing/managing_countries/province_validation.feature

```diff
@@ -17,7 +17,7 @@ Feature: Province validation
         Then I should be notified that code is required
         And province with name "Scotland" should not be added in this country
 
-    @ui @javascript
+    @ui @javascript @api
     Scenario: Trying to add a new province without specifying its name
```

Comment:
There is one missing scenario in this file: `Trying to remove name from an existing province` Do you remember, why it wasn't implemented?

Reply:
It's really an oversight, I'm going to make a new PR to add it :wink:

Reply:
Ok, I remember now :thinking:. With @Zales0123 we agreed that a sub-resource shouldn't be edited from the main resource.
However, it is exactly what we are trying to do in this scenario.
So, I don't know how to test this scenario without doing somersaults :man_dancing:

Reply:
Definitely we need to implement a PUT for the subresource. But it's also good it has not been done in this PR, as it's a little bit separated task 🖖